src/main/java/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue.java

```diff
@@ -186,4 +193,25 @@ public String toString() {
         }
         return valueStr.toString();
     }
+
+    /**
+     * return parameter description, applying the configured {@link MarkupFormatter} for jenkins instance.
+     *
+     * This is a backport from Jenkins-2.44 or Jenkins-2.32.2.
+     *
+     * @since 1.2.0
+     */
```

Comment:
I believe it's unreasonable to target Jenkins-2.46, and backported `ParameterValue.getFormattedDescription`.